src/test/fellows.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { mock, mockClear, MockProxy } from "jest-mock-extended";
+
+import { ActionLogger, TeamApi } from "../github/types";
+import { PolkadotFellows } from "../polkadot/fellows";
+
+describe("CAPI test", () => {
+  let fellows: TeamApi;
+  let logger: MockProxy<ActionLogger>;
+
+  beforeEach(() => {
+    logger = mock<ActionLogger>();
+    fellows = new PolkadotFellows(logger);
+  });
+
+  test("Should fetch fellows", async () => {
+    const members = await fellows.getTeamMembers("2");
+    expect(members.length).toBeGreaterThan(0);
+  });
+
+  test("Should cache fellows", async () => {
+    const members = await fellows.getTeamMembers("2");
+    expect(members.length).toBeGreaterThan(0);
+    expect(logger.debug).toHaveBeenCalledWith("Connecting to collective parachain");
+    mockClear(logger);
+    const members2 = await fellows.getTeamMembers("2");
+    expect(members2.length).toBeGreaterThan(0);
+    expect(logger.debug).not.toHaveBeenCalledWith("Connecting to collective parachain");
```

Comment:
this is nice test. do you plan to add the actual rule with rank:2 where you see it allows / or disallows if approver is rank < 2 ?

Reply:
That's a good question. I didn't plan on adding it, but maybe it's no trouble at all.

Reply:
It actually runs out of the box, so that's good!

Reply:
> It actually runs out of the box, so that's good!

where? I haven't seen in this diff changes where the rank: 2 approvals are being tested, could you please point my eyes?

Reply:
I added a test to [`src/test/runner/runner.test.ts`](https://github.com/paritytech/review-bot/pull/79/files#diff-22bfdc6c782b01f44253ec2a1bda5c07281909868cfcad6e4f0a79f81b30651dR58) where it verifies that the user is of a given rank and skips the test.

Reply:
hmm.. so if author is fellow - it means he doesn't need review? or I don't get
I thought they need to check whether someone how has approved - is a fellow with dan >= X, so then it counts. otherwise the review-bot blocks merge. So I was expecting this kind of test